### ISSUE TITLE
Announcements - Top Page Banner 

### ISSUE THREAD

**Issue by TamaraIDEJA** [cagov/covid19#258]
### Replace Top Page Banner "New! Hotels for Health Care Workers"
with
"Governor Newsom [announces $42 million](https://www.gov.ca.gov/2020/04/13/governor-newsom-announces-42-million-to-protect-foster-youth-and-families-impacted-by-covid-19/) to protect foster youth and families impacted by COVID19. The foster care system has been heavily impacted by COVID-19 due to school closures and social isolation. To learn how to become a foster parent and to identify the county contact in your area, visit www.cdss.ca.gov or call 1-800-KIDS-4-US (1-800-543-7487)."

### Timeline 
Today -COB 

### Dependencies

- [x] Waiting on final language
- [x] We'll have to move the current content to its new page. Angie will crate a separate Github issue for that.  New issue for this: https://github.com/cagov/covid19/issues/236 (AQ)

**Comment by abquirarte** [comment]
I made a few updates and sent over language to JIC for approval before we replace what is currently there.

**Comment by abquirarte** [comment]
This is now live. Also approved by JIC
